skills/declarative/calendarSkill/dialogs/SearchEventsDialog/language-understanding/en-us/SearchEventsDialog.en-us.lu

```diff
@@ -10,6 +10,11 @@
 - find a meeting with Title {Title=weekly report}
 - find a meeting with title {Title=second lesson} from amanda
 - find a meeting with title {Title=sharing} from steven
+- 10 am
```

Comment:
Dumb question - what is the significance of 10am in the LU model? (What about 9:30? or 11?)

Reply:
This intent is answer of "I couldn't find the event that you're looking for. Let's try your search again. Can you tell me the specific title, start date, or attendees?". So users may answer very shortly as I added. 10am has same significance with other time.

Reply:
Ah thanks!